aiida_quantumespresso/parsers/parse_raw/cp.py

```diff
@@ -204,112 +235,176 @@ def parse_cp_xml_output(data):
     parsed_data = {}
 
     #CARD HEADER
-    parsed_data = copy.deepcopy(xml_card_header(parsed_data, dom))
+    try:
+        parsed_data = copy.deepcopy(xml_card_header(parsed_data, dom))
+    except Exception as e:
+        print(e)
 
     # CARD CONTROL
 
-    cardname = 'CONTROL'
-    target_tags = read_xml_card(dom, cardname)
+    cardname='CONTROL'
+    try:
+        target_tags = read_xml_card(dom, cardname)
+    except Exception as e:
+        print(e)
+
+
+    tagname='PP_CHECK_FLAG'
+    try:
+        parsed_data[tagname.lower()] = parse_xml_child_bool(tagname, target_tags)
+    except Exception as e:
+        print(e)
 
-    tagname = 'PP_CHECK_FLAG'
-    parsed_data[tagname.lower()] = parse_xml_child_bool(tagname, target_tags)
 
     # CARD STATUS
 
     cardname = 'STATUS'
-    target_tags = read_xml_card(dom, cardname)
+    try:
+        target_tags = read_xml_card(dom,cardname)
+    except Exception as e:
+        print(e)
+
 
     tagname = 'STEP'
     attrname = 'ITERATION'
-    parsed_data[(tagname + '_' + attrname).lower()] = int(parse_xml_child_attribute_str(tagname, attrname, target_tags))
+    try:
+        parsed_data[(tagname+'_'+attrname).lower()]=int(parse_xml_child_attribute_str(tagname,attrname,target_tags))
+    except Exception as e:
+        print(e)
+
 
     tagname = 'TIME'
     attrname = 'UNITS'
-    value = parse_xml_child_float(tagname, target_tags)
-    units = parse_xml_child_attribute_str(tagname, attrname, target_tags)
-    if units not in ['pico-seconds']:
-        raise QEOutputParsingError('Units {} are not supported by parser'.format(units))
-    parsed_data[tagname.lower()] = value
+    try:
```

Comment:
Is this already parsed by the new XML parser?

Reply:
yes